File: app/src/main/java/com/example/parental_control_child/AppLauncher.java
package com.example.parental_control_child;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

public class AppLauncher extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        // Ejecutamos el panel principal
        Intent intent = new Intent(this, CodePanel.class);
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/com/example/parental_control_child/AppLauncher.kt
package com.example.parental_control_child

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class AppLauncher : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val intent = Intent(this, CodePanel::class.java)
        startActivity(intent)
    }
}